### PR TITLE
added url

### PR DESCRIPTION
**Pull request by ArvinJhurry**

### CODE REVIEW

File: wilkes/roster.json [hunk 1]
```diff
@@ -132,7 +132,9 @@
       "github": "ArvinJhurry",
       "materials": [
         {
-          "type":"cv"
+          "type":"video",
+          "url":"<https://youtu.be/FkmDhmAEVv8>"
+
         }
```
Comment (rom-30) [inline wilkes/roster.json:138]: @ArvinJhurry 
2 things
1. you need to keep the cv "object"
2. We only need to see the endpoint of the youtube video and not the full url, can you please change to "url": "FkmDhmAEVv8"

**Review by rom-30 — CHANGES_REQUESTED** [review]

File: wilkes/roster.json (hunk 1 above)
Comment (ArvinJhurry) [reply wilkes/roster.json]: Hi, I have made some changes, is this what you meant?

**Review by rom-30 — CHANGES_REQUESTED** [review]
Not quite :( 
   You need to have both  {"type":"cv" }  and {"type":"video", "url":"FkmDhmAEVv8"}. 
   Please note that I have also removed the angle brackets for the url

**Comment by ArvinJhurry** [comment]
Hi, I made another change again, i hope this is what u meant, i got a little confused aha

**Review by rom-30 — APPROVED** [review]

**Pull request merged**